ndn-svs/svspubsub.cpp

```diff
@@ -150,7 +150,7 @@ SVSPubSub::subscribeToProducer(const Name& nodePrefix, const SubscriptionCallbac
 void
 SVSPubSub::unsubscribe(uint32_t handle)
 {
-  auto unsub = [](uint32_t handle, std::vector<Subscription> subs)
+  auto unsub = [handle](std::vector<Subscription> subs)
```

Comment:
@pulsejet I'm not sure I understand this code. The lambda modifies a *copy* of the vector of subscriptions because `subs` is passed by value. Is that intended or is there a bug here?

Also, the loop uses the index instead of iterators... why?

Reply:
bug lmao. Will fix.

> Also, the loop uses the index instead of iterators... why?

No idea ...